modules/dmrpp_module/DMZ.cc

```diff
@@ -1062,17 +1173,53 @@ void DMZ::load_chunks(BaseType *btp)
     int chunks_found = 0;
     int chunk_found = 0;
     int compact_found = 0;
+
+    // Chunked data
     auto child = var_node.child("dmrpp:chunks");
     if (child) {
         chunks_found = 1;
         process_chunks(dc(btp), child);
+        auto array = dynamic_cast<Array*>(btp);
+        // It's possible to have a chunk, but not have a chunk dimension sizes element
+        // when there is only one chunk (e.g., with HDF5 Contiguous storage). jhrg 5/5/22
+        if (array && !dc(btp)->get_chunk_dimension_sizes().empty()) {
+            auto const &array_shape = get_array_dims(array);
+            size_t num_logical_chunks = logical_chunks(array_shape, dc(btp));
+            // do we need to run this code?
```

Comment:
// do we need to run this code?
Isn't the test here really for:
            // Are there missing chunks?